Makefile

```diff
@@ -6,12 +6,26 @@ PROJECT_REPO := github.com/crossplane-contrib/$(PROJECT_NAME)
 PLATFORMS ?= linux_amd64 linux_arm64
 -include build/makelib/common.mk
 
+# ====================================================================================
 # Setup Output
+
 -include build/makelib/output.mk
 
+# ====================================================================================
 # Setup Go
+
+# TODO(jastang): update Go version to be in-line with build submodule.
+GO_REQUIRED_VERSION = 1.17
```

Comment:
@chlunde i'm aware of #19 -- if we ship that first I'll rebase here and update 🙂